WebDriverAgentLib/Categories/NSExpression+FBFormat.m

```diff
@@ -10,37 +10,24 @@
 #import "NSExpression+FBFormat.h"
 #import "FBElementUtils.h"
 
-NSString *const FBUnknownPredicateKeyException = @"FBUnknownPredicateKeyException";
-
 @implementation NSExpression (FBFormat)
 
 + (instancetype)fb_wdExpressionWithExpression:(NSExpression *)input
 {
   if ([input expressionType] != NSKeyPathExpressionType) {
     return input;
   }
-  NSString *keyPath = [input keyPath];
-  NSString *actualPropName = [FBElementUtils wdAttributeNameForAttributeName:keyPath];
+  NSString *actualPropName = [input keyPath];
+  NSString *suffix = nil;
   if ([actualPropName containsString:@"."]) {
```

Comment:
Instead of querying `actualPropName` twice, you could call `[actualPropName rangeOfString:@"."]` and check location for `NSNotFound` value.